pkg/storage/tpr/install_types_test.go

```diff
@@ -148,26 +149,27 @@ func TestInstallTypesErrors(t *testing.T) {
 //make sure we don't poll on resource that was failed on install
 func TestInstallTypesPolling(t *testing.T) {
```

Comment:
@arschles I don't understand this test. I don't see how it ever worked, but apparently it must.  The part that confuses me is the use of getCallCount and comparing it to len(thirdPartyResources). That just seems really odd since we're going to Get() each resource at least once during the install (once before we install and then at least once after)  and we never reset the getCallCount so it seems like it should always generate an error 1/2 way thru the installation of the resources.